### DISCUSSION TITLE
Welcome to edgetx Discussions!

### DISCUSSION THREAD

**Discussion by raphaelcoeffic** [EdgeTX/edgetx#163] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by joevo2** [comment]
What's the plan for edgetx is there a timeline of some sort and how is it different from opentx? 

Will it support all the opentx devices? 

What's the goal and vision of edgetx? Is edgetx the likes of beta flight where as opentx will become the likes of clean flight?

**Reply by pfeerick** [reply]
> What's the plan for edgetx is there a timeline of some sort and how is it different from opentx?

Yes, 2.4.0 is slated for release on the 21st of June, 2021, and I think 2.4.1 is set for September. The current primary goal is to enable touch screen UI on colour screen radios supporting touch, as well as some other fixes along the way. It's a continuation on the OTX 2.4 that has been happening for the last 1.5 years. 
 
> Will it support all the opentx devices?

Most of the stable should be, as well as potentially some newer ones. If you have a Turnigy 9X/9XR PRO... they are basically out of luck as of around OTX 2.2 anyway... but anything from the Frsky Taranis X9D onwards should be fine... Personally for me I have it running on a Frsky X9D+,  Jumper T-Lite and RadioMaster TX16S. I know other people who have it running on the Jumper T16. 

> What's the goal and vision of edgetx? Is edgetx the likes of beta flight where as opentx will become the likes of clean flight?

That appears to be the case. I can't speak for Raphael, but I believe the primary goal is to improve the support for the current generation of transmitters. This may include improving LUA API, improving the transferability of models between transmitters by using YAML as the configuration format, improving the UI generally and making it even more configurable, adding mavlink support, just to name a few of the things being discussed or in the works.

**Reply by joevo2** [reply]
@pfeerick appreciate the explanation and thanks for the contributions to the community 😄 super excited for the release of edgetx and finally opentx 2.4. 

A bit confused on your first statement. Is touch screen support still coming with opentx 2.4 or is that a goal of edgetx instead? 

Can't wait for the merge of freedomtx with opentx and maybe finally some day I can install edgetx on my tango 2 😄

**Reply by pfeerick** [reply]
It's still the goal of OTX 2.4.0 ... but I wouldn't hold your breath... the OTX devs don't even know when OTX 2.3.12 is coming (it's currently the closest to release it's even been!!) and OTX 2.4.0 is just 'when it's ready' ... lets just leave it at that...

I would not be surprised if it were to happen... TBS are welcome to join the party ;)

**Reply by Host-Holstein** [reply]
Hello
A big request, whoever is able to include the Radioking TX18S, this radio is unfortunately not supported by openTX either.
Attempts on my Radioking TX18S the possible various relaeses for T18 T16 have failed.

**Reply by pfeerick** [reply]
TramALot beat me to to it... It seems the TX18 runs the T18 firmware, but I don't know how well it runs it... there is [mention in one issue on OTX](https://github.com/opentx/opentx/issues/8413) that the sticks don't work, and OTX [says the manufacturer never contact them](https://github.com/opentx/opentx/issues/8582), hence the lack of interest in supporting it.

**Comment by TramALot** [comment]
The current release and latest nightly work on my Radioking t18s with the
latest multi module official release. V1.3.2.86 I think

Do not put tx16 firmware on it the lcds are mounted 180, I've done it.

**Reply by pfeerick** [reply]
You didn't like the alternative operating positions and grip style that the rotated UI opened up?

**Reply by Host-Holstein** [reply]
Thank you for your prompt reply
You have to decide for yourself whether you will need it in the future or not.
I fly a lot in the sunshine, so I mainly use the Taranis x9d + display is easier to read, and I often fly with glasses.
But I also have a Jumper T16 and this RadiokingTX18S light
I only use the jumpers for my crawlers, because I can see when I'm sitting in the shade and often drink the delicious beer on the display, so I can easily imagine switching to ETX.
The Radioking is meant to be tried out, hence the attempt to fool ETX on it, I will wait and maybe there will be an explicit version for this radio after all.

Toch screen is certainly a great temptation for many, but I'm a bit conservative and am waiting for better touch displays from the manufacturers.
But it is important that it goes ahead!
Many, as I think, are very excited about what's to come.

**Reply by pfeerick** [reply]
Give it time, if we, or someone willing to write code for it, get our hands on the Radioking TX18S then it will likely get supported. It's the challenge of having both the actual hardware, and someone with the knowledge to so what's needed. Especially as having all these various transmitters ads up in $$$ quickly.

**Comment by mervin1961** [comment]


**Reply by raphaelcoeffic** [reply]
Hi! Could you please detail how you connected the touch screen? I can give you some hints also at “how to enable it”, but that will require some self-compiling ;-)

**Reply by raphaelcoeffic** [reply]
I'm not worried about your soldering skills :-D Just about how the I2C touch sensor will be connected to the radio ;-)
